docs/user-tutorial/benchmarks/micro-benchmarks.md

```diff
@@ -75,7 +75,7 @@ Inference PyTorch/ONNX models on NVIDIA GPUs with [TensorRT](https://developer.n
 #### Metrics
 
 | Name                                      | Unit      | Description                                                                                              |
-|-------------------------------------------|-----------|----------------------------------------------------------------------------------------------------------|
+| ----------------------------------------- | --------- | -------------------------------------------------------------------------------------------------------- |
 | tensorrt-inference/gpu_lat_ms_mean        | time (ms) | The mean GPU latency to execute the kernels for a query.                                                 |
 | tensorrt-inference/gpu_lat_ms_99          | time (ms) | The 99th percentile GPU latency to execute the kernels for a query.                                      |
 | tensorrt-inference/host_lat_ms_mean       | time (ms) | The mean H2D, GPU, and D2H latency to execute the kernels for a query.                                   |
```

Comment:
do we need to change "lat_ms" to "_time"?

Reply:
Yes, it's not the latency.